keps/sig-scheduling/5004-dra-extended-resource/README.md

```diff
@@ -0,0 +1,989 @@
+
+<!--
+Additionally, for Alpha try to enumerate the core package you will be touching
+to implement this enhancement and provide the current unit coverage for those
+in the form of:
+- <package>: <date> - <current test coverage>
+The data can be easily read from:
+https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit
+
+This can inform certain test coverage improvements that we want to do before
+extending the production code to implement this enhancement.
+- `<package>`: `<date>` - `<test coverage>`
+-->
+
+Start of v1.34 development cycle (v1.33.0):
+
+- `k8s.io/dynamic-resource-allocation/cel`: 88.2%
+- `k8s.io/dynamic-resource-allocation/structured`: 90.5%
+- `k8s.io/kubernetes/pkg/controller/resourceclaim`: 74.6%
+- `k8s.io/kubernetes/pkg/scheduler/framework/plugins/dynamicresources`: 65.4%
+
+##### Integration tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+-->
+
+The existing [integration tests for kube-scheduler which measure
+performance](https://github.com/kubernetes/kubernetes/tree/master/test/integration/scheduler_perf#readme)
+will be extended to cover the overheaad of running the additional logic to
+support the features in this KEP. These also serve as [correctness
+tests](https://github.com/kubernetes/kubernetes/commit/cecebe8ea2feee856bc7a62f4c16711ee8a5f5d9)
+as part of the normal Kubernetes "integration" jobs which cover [the dynamic
+resource
+controller](https://github.com/kubernetes/kubernetes/blob/294bde0079a0d56099cf8b8cf558e3ae7230de12/test/integration/scheduler_perf/util.go#L135-L139).
+
+##### e2e tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+-->
+
+End-to-end testing depends on a working resource driver and a container runtime
+with CDI support. A [test
+driver](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/dra/test-driver)
+was developed as part of the overall DRA development effort. We will add tests to
+ensure `ExtendedResourceName`s are handled by the scheduler as described in this KEP.
+
+### Graduation Criteria
+
+#### Alpha
+
+- Feature implemented behind a feature flag
+- Initial e2e tests completed and enabled
+
+#### Beta
+
+- Gather feedback
+- Additional tests are in Testgrid and linked in KEP
+
+#### GA
+
+- 3 examples of vendors making use of the extensions proposed in this KEP
+- Scalability tests that mirror real-world usage as determined by user feedback
```

Comment:
this will need to move to beta criteria based on https://github.com/kubernetes/enhancements/pull/5242

In general we are raising the bar for beta, so take a look at that and revise these criteria as needed.

Reply:
done